views/index.ejs

```diff
@@ -70,11 +70,14 @@
                                     <% } %>
                                 </div>
                                 <div class="col-4 text-end">
-                                    <div class="icon icon-shape bg-gradient-primary shadow text-center border-radius-md">
-                                        <% if (stats.icon.startsWith('fa-')) { %>
-                                            <i class="<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
-                                        <% } else { %>
-                                            <i class="ni ni-<%= stats.icon %> text-lg opacity-10" aria-hidden="true"></i>
+                                    <div class="icon icon-shape text-center border-radius-md"
+                                         style="<% if (stats.icon.startsWith('http')) { %>background-image: url('<%= stats.icon %>'); background-size: cover; background-position: center;<% } else { %>background: linear-gradient(87deg, #11cdef, #1171ef);<% } %>">
```

Comment:
if we readd above this background: linear-gradient(87deg, #11cdef, #1171ef) is not neccesarry